### PR TITLE
Login UI Component (after auth manager PR)

### PR DESCRIPTION
**Pull request by jaimedp**

### CODE REVIEW

File: src/components/login/index.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
```
Comment (narenranjit) [inline src/components/login/index.html]: We're requiring jQuery 2+ which only supports ie9+ so I think we can skip the extra checks

File: src/components/login/index.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+                <div class="control">
+                    <label for="">&nbsp;</label><button class="btn btn-primary" type="submit">Login</button>
+                </div>
+                <div id="login-message" class="alert alert-danger" style="display:none;">
+                </div>
+            </div>
+        </form>
+
+        <script src="//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
```
Comment (narenranjit) [inline src/components/login/index.html]: Let's change to jquery 2+, that's what's specified in bower.json

File: src/components/login/login.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        $('.close', dlg).off('click', null).on('click', closeDlg);
+
+        $('.group-selection-dialog').show();
+    };
+
+    $('#login, form:has(#btn-login)').on('submit', function (e) {
+        e.preventDefault();
+        var form = $(e.target);
+        var action = form.attr('action') || 'index.html';
+        var userName = $('#forio-login-username').val();
```
Comment (narenranjit) [inline src/components/login/login.js]: Let's just call it username/password etc, "forio-login-username" makes it sound like it needs to be their login for epicenter which is not necessarily true

File: src/components/login/login.js [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+        });
+    });
+
+    /* jshint multistr:true */
+    groupSelectionTemplate = window.groupSelectionTemplate = '<form>\
+        <div class="group-selection-dialog" style="display: none"> \
+            <div class="panel panel-default form-inline"> \
+                <div class="panel-heading"><span class="panel-title">Please select a group</span><span class="close pull-right">x</span></div> \
+                <div class="panel-body"> \
+                    <label for="log-to-group">Group:</label> \
```
Comment (narenranjit) [inline src/components/login/login.js]: call this "login-group" instead of "log"?

File: src/components/login/login.js [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+        e.preventDefault();
+        var form = $(e.target);
+        var action = form.attr('action') || 'index.html';
+        var userName = $('#forio-login-username').val();
+        var password = $('#forio-login-password').val();
+        var account = $('#forio-login-account').val();
+        var project = $('#forio-login-project').val();
+        var groupId = ($('#forio-login-groupId').length ? $('#forio-login-groupId') : $('<input type="hidden" id="forio-login-groupId">').appendTo(form)).val();
+
+        $('button', form).prop('disabled', 'disabled').addClass('disabled');
```
Comment (narenranjit) [inline src/components/login/login.js]: Maybe this scope this to just all buttons within itself?

File: src/components/login/index.html [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title></title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <style>
+            /* this styling is provided as an example for the html login control */
```
Comment (narenranjit) [inline src/components/login/index.html]: Why not have this in the .css?

File: src/components/login/index.html [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+
+
+            .control button:active,
+            .btn:active {
+                box-shadow: inset 1px 2px 2px rgba(0,0,0,.5);
+            }
+        </style>
+
+        <link rel="stylesheet" href="login.css">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css">
```
Comment (narenranjit) [inline src/components/login/index.html]: Maybe add a comment noting this is optional

File: src/components/login/index.html [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+    <body>
+        <form id="login">
+            <div>
+                <div class="control">
+                    <label for="">Username:</label>
+                    <input id="forio-login-username" type="text" value="jaimedp">
+                </div>
+                <div class="control">
+                    <label for="">Password:</label>
+                    <input id="forio-login-password" type="password" value="jaime123">
```
Comment (narenranjit) [inline src/components/login/index.html]: Sure about having your default password on here? :)

**Comment by jaimedp** [comment]
@narenranjit Updated PR with comments

**Pull request merged**